code/modules/mob/living/carbon/human/appearance.dm

```diff
@@ -24,12 +24,16 @@
 	var/g_eyes = 0
 	var/b_eyes = 0
 
+	var/permanently_bald = FALSE
+
 /mob/living/carbon/human/
 	var/datum/human_appearance/my_appearance
 
 /mob/living/carbon/human/proc/switch_appearance(var/datum/human_appearance/new_looks)
 	if (!istype(new_looks))
 		return
+	if (my_appearance.permanently_bald && new_looks.h_style != "Bald")
+		to_chat(src, "<span class='warning'>It is no use! The Curse O' Norwood is too strong!</span>")
```

Comment:
This doesn't prevent changing appearances

Reply:
oh yeah you're right I forgor